app/builders/application_builder.rb

```diff
@@ -59,18 +61,21 @@ def dependent_attributes(online_application)
   end
 
   def online_applicant_attributes(online_application)
-    fields = %i[title first_name last_name date_of_birth ni_number married]
+    fields = [:title, :first_name, :last_name, :date_of_birth, :ni_number, :married]
```

Comment:
I much prefer an array of symbols rather than %i etc.. - I hate them - they remind me of writing C !